dot/network/service_test.go

```diff
@@ -237,35 +237,36 @@ func TestBroadcastMessages(t *testing.T) {
 	// simulate message sent from core service
 	nodeA.GossipMessage(anounceMessage)
 	time.Sleep(time.Second * 2)
-	require.NotNil(t, handler.messages[nodeA.host.id()])
+
+	messages, _ := handler.messagesFrom(nodeA.host.id())
+	require.NotNil(t, messages)
```

Comment:
Maybe not empty? Could you also assert we get predetermined messages as well?

```suggestion
	require.NotEmpty(t, messages)
```